src/components/QuickId.vue

```diff
@@ -25,8 +25,7 @@
     },
     methods: {
       showIdManager () {
-        this.$store.dispatch('updateAllBalances')
-        this.$store.commit('toggleIdManager')
+        this.$router.push({ name: 'accounts' })
```

Comment:
The route with name `accounts` is already defined in `router/index.js`. The rest is just a link. I would like to wrap AeIdentity with RouterLink but it is impossible because AeIdentity already has a link inside. I don't see a reason to extract this handler that just set another route into vuex action. Besides vuex actions is intended to change state as a result of some asynchronous action, but route changing is not asynchronous and it doesn't mutate vuex state.